### PR TITLE
Tabledetail nunjucks macro

### PR DESCRIPTION
**Pull request by jim68000**
With magical rowspan genberator

### CODE REVIEW

**Comment by jim68000** [comment]
Hang on, don't merge. I need to add this to macro builder

**Comment by jim68000** [comment]
Ah no! It includes automatically - sweet

File: src/components/tabledetail/tabledetail.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% macro tabledetail(entries) %}
```
Comment (jim68000) [inline src/components/tabledetail/tabledetail.html]: pfft. it's self documenting code <ducks>

Comment (ztolley) [inline src/components/tabledetail/tabledetail.html]: documenting it forces you to think how it works, as well as instructing others ;) Yes I know it's simple. but all the other controls have it.

Comment (jim68000) [inline src/components/tabledetail/tabledetail.html]: Don't worry, I was joking ;)

Comment (ztolley) [inline src/components/tabledetail/tabledetail.html]: Ah yes.. humour. Oh and don't forget we code in the open.. in plain site (and government isn't supposed to have a sense of humour) ;)

**Comment by jim68000** [comment]
I've added the gallery page, but we should probably park this so I can refactor it into a &lt;dl>

**Pull request closed without merging**